.ci/kitchen-centos6-py2

```diff
@@ -29,209 +30,96 @@ if (buildNumber > 1) {
-    chunks[stagename] = run_tests(checkout_directory, stagename, env_array, 2)
-}
-
-// Integration State Tests
-def integration_states_chunks = 2
-for (int i=1; i<(integration_states_chunks+1); i++) {
-    def chunk_no = i
-    def stagename = "Integration States #${chunk_no}"
-    def env_array = [
-        "NOX_PASSTHROUGH_OPTS=${nox_passthrough_opts} --test-group-count=$integration_states_chunks --test-group=$chunk_no tests/integration/states"
-    ]
-    chunks[stagename] = run_tests(checkout_directory, stagename, env_array, 2)
-}
-
-// Unit Tests
-def unit_chunks = 2
-for (int i=1; i<(unit_chunks+1); i++) {
-    def chunk_no = i
-    def stagename = "Unit #${chunk_no}"
-    def env_array = [
-        "NOX_PASSTHROUGH_OPTS=${nox_passthrough_opts} --test-group-count=$unit_chunks --test-group=$chunk_no tests/unit"
-    ]
-    chunks[stagename] = run_tests(checkout_directory, stagename, env_array, 2)
-}
-
-// All Other
-def other_chunks = 4
-for (int i=1; i<(other_chunks+1); i++) {
-    def chunk_no = i
-    def stagename = "All Other #${chunk_no}"
-    def env_array = [
-        "NOX_PASSTHROUGH_OPTS=${nox_passthrough_opts} --test-group-count=$other_chunks --test-group=$chunk_no --ignore=tests/integration/modules --ignore=tests/integration/states --ignore=tests/unit"
-    ]
-    chunks[stagename] = run_tests(checkout_directory, stagename, env_array, 2)
-}
-
-
-node('kitchen-xl-slave') {
-    timeout(time: global_timeout, unit: 'HOURS') {
-        withCredentials([[$class: 'AmazonWebServicesCredentialsBinding',
-                          accessKeyVariable: 'AWS_ACCESS_KEY_ID',
-                          credentialsId: 'AWS_ACCESS_KEY_ID',
-                          secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+node('kitchen-slave') {
+    withSetBuildResult("#jenkins-prod-pr") {
+        timeout(time: global_timeout, unit: 'HOURS') {
             ansiColor('xterm') {
                 timestamps {
                     withEnv([
                         'SALT_KITCHEN_PLATFORMS=/var/jenkins/workspace/nox-platforms.yml',
                         'SALT_KITCHEN_VERIFIER=/var/jenkins/workspace/nox-verifier.yml',
                         'SALT_KITCHEN_DRIVER=/var/jenkins/workspace/driver.yml',
-                        'NOX_ENV_NAME=pytest-zeromq',
+                        "NOX_ENV_NAME=pytest-${test_transport.toLowerCase()}",
                         'NOX_ENABLE_FROM_FILENAMES=true',
+                        "NOX_PASSTHROUGH_OPTS=${nox_passthrough_opts}",
                         "SALT_TARGET_BRANCH=${salt_target_branch}",
                         "GOLDEN_IMAGES_CI_BRANCH=${golden_images_branch}",
-                        "CODECOV_FLAGS=${distro_name}${distro_version},${python_version}",
+                        "CODECOV_FLAGS=${distro_name}${distro_version},${python_version},${test_transport.toLowerCase()}",
                         'PATH=~/.rbenv/shims:/usr/local/rbenv/shims:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin:/root/bin',
```

Comment:
at some point, we should remove the /usr/local/rbenv/shims since that's not used any more, and the extra /root/bin on the end.

Reply:
Now is good

Reply:
Done

Reply:
Done